src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,109 @@
+import { NavLink, useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const PeopleTable = () => {
+  const { personSlug } = useParams();
+
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [people, setPeople] = useState<Person[] | []>([]);
+  const [error, setError] = useState<boolean>(false);
+
+  const peopleSuccessfulLoaded = !error && people.length > 0 && !isLoading;
+  const noPeopleFound = !error && people.length === 0 && !isLoading;
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
if you apply the logic above, you could remove this line

Reply:
done ;)